master/docs/examples/git_gerrit.cfg

```diff
@@ -163,7 +166,9 @@ def gerritSummaryCB(buildInfoList, results, status, arg):
             msg += " - " + link
         else:
             msg += "."
-        msgs.append(msg)
+
+        if buildInfo['build']['properties']['shouldReport']:
```

Comment:
Agreed. I want to demonstrate that the properties are available and can be used. Do you think that's necessary? If so, how would you recommend exemplifying that?